src/__tests__/testScreen.py

```diff
@@ -92,6 +92,17 @@
         'maxX': 20,
         'maxY': 30,
     }
+}, {
+    'name': 'dontWipeChrome',
+    'input': 'gitDiffColor.txt',
+    'withAttributes': True,
+    'validatesFileExists': False,
+    'inputs': ['DOWN', 'f', 'f', 'f', 'UP'],
+    'screenConfig': {
+        'maxX': 201,
+        'maxY': 40
+    },
+    'pastScreen': [0, 1, 2, 3, 4]
```

Comment:
oh I see. I'd rather have this be something like `pastScreens` rather than overloading the type of `pastScreen` and allowing lists as well as single screens.

that way the test checker can just become a loop of what we had earlier